src/BlazorUI/Bit.BlazorUI/Styles/Fluent/colors.fluent-dark.scss

```
@@ -176,7 +208,23 @@
     --bit-clr-brd-ter-light: #2E343E;
     --bit-clr-brd-ter-light-hover: #505B6F;
     --bit-clr-brd-ter-light-active: #637189;
+    --bit-clr-brd-ter-dis: #1A1F27;
+    --bit-clr-brd-ter-dis-text: #2A3038;
     --bit-clr-brd-dis: #21262D;
     //req
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)